__tests__/app.test.js

```diff
@@ -0,0 +1,49 @@
+const { app } = require('../app')
+const request = require('supertest')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+
+const testData = require('../db/data/test-data')
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    db.end();
+})
+
+describe("GET /api/topics", () => {
+    it("returns an array of objects", () => {
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                expect(Array.isArray(response.body)).toBe(true)
+                const body = response.body
+                body.forEach((obj) => {
+                    expect(obj === Object(obj)).toBe(true)
+                })
+            })
+    })
+    it("topic objects contain the slug & description properties", () => {
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                const body = response.body
```

Comment:
As we are using a forEach here - if the array is empty then this test will pass by default as none of the expect statements will be called. A test with no expect statements will pass by default. We should first check that there is something in the array (check its length)